pkg/cmdutils/token.go

```diff
@@ -37,8 +37,8 @@ type Token interface {
 	CreateSecretKey(algorithm.Algorithm) ([]byte, error)
 
 	// Create creates a token object using the specified signature algorithm, private key,
-	// object and the expire time
-	Create(algorithm.Algorithm, interface{}, string, int64) (string, error)
+	// object and the expire time and header
+	Create(algorithm.Algorithm, interface{}, string, int64, map[string]interface{}) (string, error)
```

Comment:
why not `map[string]string`?

Reply:
In order to be consistent with the definition of Jwt Token

```go
// Token represents a JWT Token.  Different fields will be used depending on whether you're
// creating or parsing/verifying a token.
type Token struct {
	Raw       string                 // The raw token.  Populated when you Parse a token
	Method    SigningMethod          // The signing method used or to be used
	Header    map[string]interface{} // The first segment of the token
	Claims    Claims                 // The second segment of the token
	Signature string                 // The third segment of the token.  Populated when you Parse a token
	Valid     bool                   // Is the token valid?  Populated when you Parse/Verify a token
}
```